pkg/reconciler/pingsource/pingsource_test.go

```diff
@@ -141,15 +137,14 @@ func TestAllCases(t *testing.T) {
 						Data:     testData,
 						Sink:     &sinkDest,
 					}),
-					WithPingSourceResourceScopeAnnotation,
 					WithPingSourceUID(sourceUID),
 					WithPingSourceObjectMetaGeneration(generation),
 				),
 				NewChannel(sinkName, testNS,
 					WithInitChannelConditions,
 					WithChannelAddress(sinkDNS),
 				),
-				makeAvailableReceiveAdapter(sinkDest),
+				makeAvailableMTAdapter(),
```

Comment:
there is only one - right ? 

should we (in a different PR) change that MT vs ST naming here ?